server/web/public/scripts/proposals/submissions-list.js

```diff
@@ -19,3 +19,50 @@ function updateFeasibilityStatus(proposalId, approved) {
     },
   });
 }
+
+function openFeasibilityModal(proposalId){
+  const modal = document.getElementById('feasibility-modal-' + proposalId);
+  modal.style.display = "block"; 
+}
+
+function closeFeasibilityModal(proposalId) {
+  const modal = document.getElementById('feasibility-modal-' + proposalId);
+  modal.style.display = "none";
+}
+
+ // When the user clicks on the button, open the modal
+ function openModal(proposalId) {
+  
+  const modal = document.getElementById('modal-' + proposalId);
+  modal.style.display = "block"; 
+}
+
+function assignReviewer(proposalId){
+  const modalVal = document.getElementById('reviewerSelect');
```

Comment:
We're using jQuery across anchor code base, need to refactor all this in jQuery.